<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AllcarsDialog</class>
 <widget class="QDialog" name="AllcarsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>564</width>
    <height>303</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog
{
	background-color:#353935;
}</string>
  </property>
  <widget class="QPushButton" name="backbutton">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>250</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="carnameline">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>40</y>
     <width>121</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit
{
	border:0;
	color:white;
	background-color:transparent;
}</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>52</x>
     <y>40</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit
{
	border:0;
	color:white;
	background-color:transparent;
	font: 700 italic 10pt &quot;Segoe UI&quot;;
}</string>
   </property>
   <property name="text">
    <string>Your car is :</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>50</y>
     <width>191</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="audia4label">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>70</y>
     <width>281</width>
     <height>141</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>D:/test/audia4.jpg</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="audia6_label">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>70</y>
     <width>281</width>
     <height>141</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>D:/test/audi_a6.jpg</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="audia7_label">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>70</y>
     <width>281</width>
     <height>141</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>D:/test/audia7.jpg</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="bmw3er_label">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>70</y>
     <width>281</width>
     <height>141</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>D:/test/bmw3er.jpg</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="bmw5er_label">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>70</y>
     <width>281</width>
     <height>141</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>D:/test/bmw5er.jpg</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="bmw7er_label">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>70</y>
     <width>281</width>
     <height>141</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>D:/test/bmw7er.jpg</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
